src/pc_screen_effect.c

```diff
@@ -9,10 +9,10 @@
 #define tState data[0]
 #define tXSpeed data[1]
 #define tYSpeed data[2]
-#define tWin0Left data[3]
-#define tWin0Right data[4]
-#define tWin0Top data[5]
-#define tWin0Bottom data[6]
+#define twirelessWinLeft data[3]
+#define twirelessWinRight data[4]
```

Comment:
These look like a mistake from a global find and replace.

Reply:
Made the Changes and also some more stuff

Reply:
Still someone need to fully write down map script file comment info specific to FRLG